### ISSUE TITLE
Compile test broken

### ISSUE THREAD

**Issue by racke** [itnode/DBIx-Class-Elasticsearch#7]
#   Failed test 'use DBIx::Class::Elasticsearch;'
```
#   at t/00_compile.t line 4.
#     Tried to use 'DBIx::Class::Elasticsearch'.
#     Error:  Can't locate DBIx/Class/Elasticsearch.pm in @INC (you may need to install the DBIx::Class::Elasticsearch module) (@INC contains: /home/racke/.cpanm/work/1449058682.17407/DBIx-Class-ElasticSync-0.03/blib/arch /home/racke/.cpanm/work/1449058682.17407/DBIx-Class-ElasticSync-0.03/blib/lib /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.2/x86_64-linux /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.2 /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/5.20.2/x86_64-linux /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/5.20.2 /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.0/x86_64-linux /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.0 /home/racke/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl .) at t/00_compile.t line 4.
# BEGIN failed--compilation aborted at t/00_compile.t line 4.
# Looks like you failed 1 test of 1.
t/00_compile.t .. 
```

**Comment by karenetheridge** [comment]
also: https://rt.cpan.org/Ticket/Display.html?id=110026

**Comment by m0veax** [comment]
fixed with 437238c9e8e60359328dbed52edeb1643318957f

Thanks for reporting.
